packages/test/test-service-load/package.json

```diff
@@ -59,12 +59,12 @@
 		"cache-dir": "nyc/.cache",
 		"exclude": [
 			"src/test/**/*.*ts",
-			"dist/test/**/*.*js"
+			"lib/test/**/*.*js"
```

Comment:
Lots of things left using dist in here. See package.json scripts, .mocharc.cjs, and src/test/mini.spec.ts that launches `./dist/nodeStressTest.js`.

Reply:
Switched c8 back to /dist/ and created AB#7443 to track fully converting to ESM.